### PR TITLE
#24 - Successfully getting all likes

### PR DESCRIPTION
**Pull request by jabui**
Wasn't sure how to commit to my previous pull request.

Using Pagination, must have overlooked it the first time i went through the docs ~  [Soundcloud API] (https://developers.soundcloud.com/docs/api/reference#activities)

An argument to limit the number of results returned might be necessary.

This is only for downloading all 'likes', i didn't take into account if a user had more than 200 tracks on their profile. I'm pretty new to Python, so feedback welcome :smile:

### CODE REVIEW

**Comment by tyhallcsu** [comment]
This is very nice! Tested and it allows me to download more likes than the (undefined) 200 limit. Will get back in a few and let you know if the script traversed all my 750 liked tracks. But overall, looks good so far

**Comment by tyhallcsu** [comment]
`The Presets - Pacifica Deluxe Mix is not downloadable.`
Link:
https://soundcloud.com/modularpeople/the-presets-pacifica-deluxe

`Love Sosa (RL Grime Remix) - Chief Keef is not downloadable.`
Link:
https://soundcloud.com/rlgrime/love-sosa-rl-grime-remix

`01. LOVE YOU is not downloadable.`
Link:
https://soundcloud.com/inglewoodsir/01-love-you


```
Downloading: Die For Me (feat. KYLE & Christoph Andersson)
[################################] 9321/9321 - 00:00:05
can't sync to an MPEG frame
```
Link:
https://soundcloud.com/marty-grimes-music/die-for-me-feat-kyle-christoph-andersson

File: soundscrape/soundscrape.py [hunk 1]
```diff
@@ -16,6 +17,12 @@
 from subprocess import Popen, PIPE
 from os.path import exists, join
 from os import mkdir
+try:
+    # For Python 3.0 and later
+    from urllib.request import urlopen
```
Comment (pydo) [inline soundscrape/soundscrape.py]: Why are you using urllib when requests is already being used?

**Comment by Miserlou** [comment]
Merged in manually, thanks! Sorry for the delay.

**Pull request closed without merging**